source/algorithms.tex

```diff
@@ -1434,33 +1434,35 @@
     template<class I, class O>
     using replace_copy_result = copy_result<I, O>;
 
-    template<InputIterator I, Sentinel<I> S, class T1, class T2, OutputIterator<const T2&> O,
+    template<input_iterator I, sentinel_for<I> S,
+             class T1, class T2, output_iterator<const T2&> O,
              class Proj = identity>
```

Comment:
Any reason not to two-line this *template-head* a la:
```c++
template<input_iterator I, sentinel_for<I> S, class T1, class T2, 
         output_iterator<const T2&> O, class Proj = identity>
```

Reply:
Was trying to create some consistency with how other params were wrapped, but I like your suggestion - fixed.